src/react/src/components/SiteNav/SiteNav.js

```diff
@@ -15,6 +15,11 @@ function SiteNav(props) {
             Links
           </Link>
         </li>
+        <li>
+          <Link to="/lists" className="drizzle-b-Link drizzle-b-Link--light">
```

Comment:
lets use list instead of lists, since we decided to be singular with everything

Reply:
Should I update the rest of the naming in the file? Everything else is plural in the nav.